### PR TITLE
fix container name in addsecuritycontext patch

### PR DESCRIPTION
**Pull request by lybavsky**

### CODE REVIEW

File: pkg/webhook/patch.go [hunk 1]
```diff
@@ -613,14 +613,9 @@ func addSecurityContext(pod *corev1.Pod, app *v1beta2.SparkApplication) *patchOp
...
-	// Find the driver/executor container in the pod.
-	for ; i < len(pod.Spec.Containers); i++ {
-		if pod.Spec.Containers[i].Name == config.SparkDriverContainerName || pod.Spec.Containers[i].Name == config.SparkExecutorContainerName {
-			break
-		}
-	}
-	if i == len(pod.Spec.Containers) {
+	i := findContainer(pod)
+
+	if i == -1 {
```
Comment (liyinan926) [inline pkg/webhook/patch.go:618]: Nit: can we use `i < 0` here instead?

Comment (lybavsky) [reply pkg/webhook/patch.go:618]: I think yes, anyway findContainer returns only non-negative values if container found in pod, or -1 if not.

Comment (liyinan926) [reply pkg/webhook/patch.go:618]: Yes, but I think it's safer to check `i < 0`.

Comment (lybavsky) [reply pkg/webhook/patch.go:618]: Ok, sure,this is acceptable

Comment (lybavsky) [reply pkg/webhook/patch.go]: Fixed!

**Pull request merged**